### PR TITLE
Feature/7707 jdbc types

### PR DESCRIPTION
**Pull request by alexradzin**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -38,5 +40,5 @@ dependencies {
     compileOnly "org.postgresql:postgresql:42.2.9"
     compileOnly "com.microsoft.sqlserver:mssql-jdbc:8.2.0.jre11"
 
-//    implementation fileTree(dir: '/tmp/p/com/upsolver/data-sources-interfaces/1.0.1/', include: '*.jar')
+    implementation fileTree(dir: '/tmp/p/com/upsolver/data-sources-interfaces/1.0.5-SNAPSHOT/', include: '*.jar')
```
Comment (jasonf20) [inline build.gradle]: revert before merging.

File: build.gradle [hunk 2]
```diff
@@ -28,8 +28,10 @@ repositories {
 dependencies {
     compile 'com.zaxxer:HikariCP:3.4.2'
     implementation 'org.slf4j:slf4j-api:1.7.30'
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.11.2'
     testCompile group: 'junit', name: 'junit', version: '4.12'
-    implementation 'com.upsolver:data-sources-interfaces:1.0.6'
+//    implementation 'com.upsolver:data-sources-interfaces:1.0.7'
```
Comment (jasonf20) [inline build.gradle]: revert before merging

File: src/main/java/com/upsolver/datasources/jdbc/JDBCDataSource.java [hunk 3]
```diff
@@ -67,14 +69,31 @@
...
+                    new SimplePropertyDescription(fullLoadIntervalProp, "If set the full table will be read every configured interval (in minutes). When this is configured the update time and incrementing columns are not used.", true),
+                    new SimplePropertyDescription(keepSourceTypes, "Keep original data types from source to use string representation", true, false, null, null, null, true, Optional.of("true")));
 
     private long readDelay;
     private long fullLoadIntervalMinutes;
     private TableInfo tableInfo;
     private QueryDialect queryDialect;
     private long dbTimezoneOffset;
     private long overallQueryTimeAdjustment;
+    boolean keepTypes = false;
```
Comment (jasonf20) [inline src/main/java/com/upsolver/datasources/jdbc/JDBCDataSource.java]: private

File: build.gradle (hunk 1 above)
Comment (alexradzin) [reply build.gradle]: sure. I know I have to revert this.

File: src/main/java/com/upsolver/datasources/jdbc/JDBCDataSource.java (hunk 3 above)
Comment (alexradzin) [reply src/main/java/com/upsolver/datasources/jdbc/JDBCDataSource.java]: done

File: build.gradle (hunk 2 above)
Comment (alexradzin) [reply build.gradle]: done

File: build.gradle (hunk 1 above)
Comment (alexradzin) [reply build.gradle]: done

**Review by jasonf20 — APPROVED** [review]

**Pull request merged**